### PR TITLE
fix(Content): ul li color in dark theme

### PR DESCRIPTION
**Pull request by yuberdysheva**

### CODE REVIEW

File: src/sub-blocks/BackgroundCard/__stories__/BackgroundCard.stories.tsx [hunk 1]
```diff
@@ -46,7 +47,15 @@ const CardThemesTemplate: Story<BackgroundCardProps> = (args) => (
             <BackgroundCard {...args} title="Тема по умолчанию = s" theme="default" />
         </div>
         <div style={{maxWidth: '400px', padding: '0 8px'}}>
-            <BackgroundCard {...args} title="Темная тема = m" theme="dark" />
+            <BackgroundCard
+                {...args}
+                title="Темная тема = m"
```
Comment (gorgeousvlad) [inline src/sub-blocks/BackgroundCard/__stories__/BackgroundCard.stories.tsx:52]: all text constants must be placed in data.json
no russian - english only

File: src/sub-blocks/BackgroundCard/__stories__/BackgroundCard.stories.tsx [hunk 2]
```diff
@@ -46,7 +47,15 @@ const CardThemesTemplate: Story<BackgroundCardProps> = (args) => (
...
+                theme="dark"
+                background={{
+                    ...args.buttons,
+                    src: 'https://storage.yandexcloud.net/cloud-www-assets/constructor/storybook/images/img-black.png',
```
Comment (gorgeousvlad) [inline src/sub-blocks/BackgroundCard/__stories__/BackgroundCard.stories.tsx:56]: can we place link in data.json?

**Review by gorgeousvlad — CHANGES_REQUESTED** [review]

File: src/sub-blocks/BackgroundCard/__stories__/BackgroundCard.stories.tsx (hunk 1 above)
Comment (yuberdysheva) [reply src/sub-blocks/BackgroundCard/__stories__/BackgroundCard.stories.tsx]: ok

File: src/sub-blocks/BackgroundCard/__stories__/BackgroundCard.stories.tsx (hunk 2 above)
Comment (yuberdysheva) [reply src/sub-blocks/BackgroundCard/__stories__/BackgroundCard.stories.tsx]: ok

File: package.json [hunk 3]
```diff
@@ -121,5 +121,8 @@
     "*.{json,yaml,yml,md}": [
       "prettier --write"
     ]
+  },
+  "publishConfig": {
+    "tag": "alpha"
```
Comment (gorgeousvlad) [inline package.json:126]: please remove it until latest version setting will be added to action
this way every new version will publish like alpha

**Review by gorgeousvlad — APPROVED** [review]

**Review by aeksandla — APPROVED** [review]

**Pull request merged**